terraform/monitoring/dashboards/adservice_dashboard.json

```diff
@@ -0,0 +1,124 @@
+{
+  "displayName": "Ad Service Dashboard",
+  "gridLayout": {
+    "columns": "2",
+    "widgets": [
+      {
+        "title": "CPU limit utilization for stackdriver-sandbox, adservice",
+        "xyChart": {
+          "dataSets": [
+            {
+              "timeSeriesQuery": {
+                "timeSeriesFilter": {
+                  "filter": "metric.type=\"kubernetes.io/container/cpu/limit_utilization\" resource.type=\"k8s_container\" resource.label.\"cluster_name\"=\"stackdriver-sandbox\" metadata.user_labels.\"app\"=\"adservice\" resource.label.\"container_name\"=\"server\"",
+                  "aggregation": {
+                    "perSeriesAligner": "ALIGN_MEAN"
+                  },
+                  "secondaryAggregation": {}
```

Comment:
I think empty secondaryAggregation objects and default timeshiftDuration fields should be removed to keep this examples as clean as possible. See original comment here: https://github.com/GoogleCloudPlatform/stackdriver-sandbox/pull/175#discussion_r460247643

Reply:
Yup! Changed :)